spk/itools/src/installer.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
+
+# Package
+PACKAGE="itools"
+DNAME="itools"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Put tmux in the PATH
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/bin/idevicebackup /usr/local/bin/idevicebackup
```

Comment:
I find all these symbolic links "ugly". Commands are easily accessible from `/usr/local/itools/bin` if needed and this location may be added to PATH. I would only create links for command end-user may really invoke himself.

Reply:
For link creation, you can inspire from recent PR: https://github.com/SynoCommunity/spksrc/pull/3378/files